app/common/patient/models/patient.js

```diff
@@ -16,6 +16,15 @@ angular.module('common.patient')
                 this.birthdate = age.calculateBirthDate(this.age);
             };
 
+            var calculateBirthdateEstimated = function (value) {
+                if (value === 'age') {
```

Comment:
willl this work with any locale apart form en? or this is not affected with translation